### PR TITLE
Fix `browse_website` headless mode for Firefox 

### PR DESCRIPTION
**Pull request by Irmius**
make changes to work docker with mac m1 chip browse_website command as chromium drivers is crashing and need to use firefox. Update README.md

### PR Quality Checklist
- [x] My pull request is atomic and focuses on a single change.
- [x] I have thoroughly tested my changes with multiple different prompts.
- [x] I have considered potential risks and mitigations for my changes.
- [x] I have documented my changes clearly and comprehensively.
- [x] I have not snuck in any "extra" small tweaks changes

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ services:
     env_file:
       - .env
     volumes:
+      - "./auto_gpt_workspace:/home/appuse/auto_gpt_workspacer"
```
Comment (Wladastic) [inline docker-compose.yml]: workspacer?

File: docker-compose.yml [hunk 2]
```diff
@@ -10,6 +10,7 @@ services:
     env_file:
       - .env
     volumes:
+      - "./auto_gpt_workspace:/home/appuser/auto_gpt_workspace"
```
Comment (k-boikov) [inline docker-compose.yml]: I'm afraid that would be something for another PR.

**Review by Pwuts — APPROVED** [review]
Tested manually by gNoLEm on Discord, thanks! <3
